src/main/java/com/exadel/discountwebapp/category/vo/CategoryRequestVO.java

```diff
@@ -0,0 +1,19 @@
+package com.exadel.discountwebapp.category.vo;
+
+import com.exadel.discountwebapp.category.entity.Category;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class CategoryRequestVO extends CategoryBaseVO {
+
+    public static Category toCategory(CategoryRequestVO request) {
```

Comment:
VO classes should not contain any logic. This mapping logic should be moved to CategoryMapper.
It will have 2 methods:
toVO(Category entity)
toEntity(CategoryRequestVO requestVO)

Reply:
done